R/fetch.R

```diff
@@ -19,11 +21,14 @@ git_fetch <- function(remote = NULL, refspec = NULL, password = askpass,
                       ssh_key = NULL, verbose = interactive(), repo = '.'){
   repo <- git_open(repo)
   info <- git_info(repo)
+
+  remote <- remote %||% info$remote
```

Comment:
I don't think we have `%||%`

Reply:
Edit: oh wait I see it now at the bottom :)

Reply:
Yeah, I can remove that and spell it out if you wish. But I use this operator all the time (I usually take it from rlang).

Reply:
Yes I prefer to use base-r syntax like in the original code. I always use this pattern:

```r
  if(!length(remote))
   remote <- info$remote
```